data/strings/strings.txt

```diff
@@ -10433,6 +10433,13 @@
     zh-Hans = 添加街道
     zh-Hant = 新增街道
 
+  [empty_street_name_error]
+    comment = New street dialog validation message
```

Comment:
Error to display when a new street name is not entered in the New street dialog.

Reply:
Updated comment